securityscanutils/predicate_test.go

```diff
@@ -34,9 +34,6 @@ var _ = Describe("Predicate", func() {
 			Entry("release is draft", &github.RepositoryRelease{
 				Draft: github.Bool(true),
 			}, false),
-			Entry("release is pre-release", &github.RepositoryRelease{
```

Comment:
I think we still need this test, right? This test is used to demonstrate that github pre-releases are included